src/code-snippet/code-snippet.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+
+import { CodeSnippetModule } from "..";
+
+
+const code = `import { storiesOf, moduleMetadata } from "@storybook/angular";
```

Comment:
start the code maybe on the next line? Same with `inlineCode`

Reply:
That new line will then be in the snippet, not what we want.